packages/pytest-simcore/src/pytest_simcore/helpers/s3.py

```diff
@@ -100,7 +100,7 @@ async def upload_file_to_presigned_link(
                         upload_url,
                     )
                 )
-            results = await logged_gather(*upload_tasks, max_concurrency=0)
+            results = await logged_gather(*upload_tasks, max_concurrency=20)
```

Comment:
could it not be repalced with `limited_gather`  here?

Reply:
probably.